docs-ref-conceptual/authenticate-azure-cli.md

```diff
@@ -59,14 +59,18 @@ az login -u <username> -p <password>
 Service principals are accounts not tied to any particular user, which can have permissions on them assigned through
 pre-defined roles. Authenticating with a service principal is the best way to write secure scripts or programs,
 allowing you to apply both permissions restrictions and locally stored static credential information. To learn more
-about service principals, see [Create an Azure service principal with the Azure CLI](create-an-azure-service-principal-azure-cli.md).
+about service principals, see [Create an Azure service principal with the Azure CLI](/cli/azure/create-an-azure-service-principal-azure-cli.md).
```

Comment:
However I still think this is something that should be addressed in the build pipeline (or CI), since using absolute link makes navigating in GitHub impossible. For example, in this [preview page](https://github.com/dbradish-microsoft/azure-docs-cli/blob/34c79bf72a997457d93f1cde76f59cb871eb71e0/docs-ref-conceptual/authenticate-azure-cli.md#sign-in-with-a-service-principal), if I click [Create an Azure service principal with the Azure CLI](https://github.com/dbradish-microsoft/azure-docs-cli/blob/34c79bf72a997457d93f1cde76f59cb871eb71e0/cli/azure/create-an-azure-service-principal-azure-cli.md), it returns a 404 error.